src/main/java/io/moderne/organizations/OrganizationStructureService.java

```diff
@@ -40,67 +43,65 @@ public Map<String, OrganizationRepositories> readOrganizationStructure() {
 
         final Map<String, String> idToNameMapping = readIdToNameMapping();
 
-        InputStream inputStream;
-        try {
-            if (reposCsvPath == null) {
-                inputStream = new ClassPathResource(DEFAULT_REPOS_CSV).getInputStream();
-            } else {
-                inputStream = new FileInputStream(reposCsvPath.toFile());
-            }
-        } catch (IOException e) {
-            throw new UncheckedIOException(e);
+        CsvMapper csvMapper = new CsvMapper();
+        CsvSchema bootstrapSchema = CsvSchema.emptySchema()
+                .withComments()
+                .withHeader();
+
+        File inputFile;
+        if (reposCsvPath == null) {
+            inputFile = new File(DEFAULT_REPOS_CSV);
```

Comment:
I dont think this will work when the resource is inside the jar. You will need to use an input stream (which should be fine with jackson)

Reply:
Will update to use an `InputStream` instead of a `File`